dm/loader/lightning.go

```diff
@@ -229,14 +234,18 @@ func (l *LightningLoader) restore(ctx context.Context) error {
 		}
 
 		cfg.TiDB.StrSQLMode = l.cfg.LoaderConfig.SQLMode
-		cfg.TiDB.Vars = map[string]string{
-			"time_zone": l.timeZone,
-		}
+		cfg.TiDB.Vars = map[string]string{"time_zone": l.timeZone}
 		if err = cfg.Adjust(ctx); err != nil {
 			return err
 		}
 		err = l.runLightning(ctx, cfg)
 		if err == nil {
+			// lightning will auto deregister tls when task done, so we need to register it again for removing checkpoint
```

Comment:
we might let lightning not deregister the certificate. If there're more than one subtasks, the certificate may still be deregister between line 245 and line 249

Reply:
now lightning allow  register tls config with different name, and it test is passed, ptal 🧡

Reply:
CheckpointRemove should be postponed to cleanmeta. I will add this logic in a following PR, so LGTM.